src/main/java/binaries/alejandro/BinaryArray.java

```diff
@@ -18,6 +18,10 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+	    int result = 0;
```

Comment:
Please use 4 spaces instead of tabs. You should be able to configure this in your editor settings